src/TTextEdit.cpp

```diff
@@ -1238,17 +1238,22 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
         // in the QAction constructor:
         action->setToolTip(QString());
         connect(action, SIGNAL(triggered()), this, SLOT(slot_copySelectionToClipboard()));
-        QAction* action2 = new QAction("copy HTML", this);
+        QAction* action2 = new QAction(tr("Copy HTML"), this);
         action2->setToolTip(QString());
         connect(action2, SIGNAL(triggered()), this, SLOT(slot_copySelectionToClipboardHTML()));
-        QAction* action3 = new QAction("select all", this);
+        QAction* action3 = new QAction(tr("Select All"), this);
         action3->setToolTip(QString());
         connect(action3, SIGNAL(triggered()), this, SLOT(slot_selectAll()));
 
         QString selectedEngine = mpHost->getSearchEngine().first;
-        QAction* action4 = new QAction(("search on " + selectedEngine), this);
+        QAction* action4 = new QAction(tr("Search on %1").arg(selectedEngine), this);
         action4->setToolTip(QString());
         connect(action4, SIGNAL(triggered()), this, SLOT(slot_searchSelectionOnline()));
+        if (!qApp->testAttribute(Qt::AA_DontShowIconsInMenus)) {
+            action->setIcon(QIcon::fromTheme(QStringLiteral("edit-copy"), QIcon(QStringLiteral(":/icons/edit-copy.png"))));
+            action3->setIcon(QIcon::fromTheme(QStringLiteral("edit-select-all"), QIcon(QStringLiteral(":/icons/edit-select-all.png"))));
+            action4->setIcon(QIcon::fromTheme(QStringLiteral("edit-web-search"), QIcon(QStringLiteral(":/icons/edit-web-search.png"))));
```

Comment:
This icon looks pretty poor, I'd rather do with a different one or without it.

Reply:
It's what I pulled from my goto icon source - the Oxygen set from my Debian distribution -- however by virtue of the `QIcon::fromTheme` are you seeing that one in the resources file or one from your own Desktop icon theme selection?

Reply:
Yeah I'm seeing that one, the `edit-web-search.png`

Reply:
You mean: 
![edit-web-search](https://user-images.githubusercontent.com/6163092/42508575-e5ed8dc2-8440-11e8-9cf0-0ba2fbbf2514.png)

https://github.com/pasnox/oxygen-icons-png/blob/master/oxygen/48x48/actions/edit-web-search.png

Well it is the themed (so can be replaced on other Linux distributions) icon which seems most appropriate for web (the "World") searches (the "Binoculars") - BTW the latter **does** appear elsewhere within our application in relation to searching operations.  Given that you do not wish to have such icons you can of course now switch it and all the others off - but is it really *that* bad? :confused: 

*\<off-topic> It does make more sense to me than the pair of people as the "Aliases" one: 
![system-users](https://user-images.githubusercontent.com/6163092/42512360-587045d2-844c-11e8-8228-5c55c7cb76a0.png)\</off-top>*

Reply:
I don't think this particular icon is used anywhere else! Can you show me where?

Reply:
We haven't used it in Mudlet in the past but it seemed a reasonable one to use. It looks as though it was adopted into KDE in 2011. https://git.reviewboard.kde.org/r/100608/

Reply:
Yeah, I still think it looks kind of terrible, and it really is from a decade ago - not modern at all. But I'll drop this, can't have everything be perfect.